ios/FluentUI/Notification/MSFNotificationView.swift

```diff
@@ -59,7 +59,7 @@ import UIKit
         let presentationOffset: CGFloat! = notification.tokens.presentationOffset
         if style.isToast, let currentToast = MSFNotification.currentToast, currentToast.window != nil {
             currentToast.hide {
-                self.showNotification(in: view, completion: completion)
```

Comment:
is there still use of self.shownotification(in: completion)?

Reply:
Only one left is in the demo controller.